docs/reference/flowdecomposition.rst

```diff
@@ -0,0 +1,18 @@
+Flow decomposition
+==================
+
+.. module:: pypowsybl.flowdecomposition
+
+The flow decomposition module allows to decompose flows on cross-border relevant network element with contingency (XNEC) based on the ACER methodology.
```

Comment:
active and reactive flows? Can you explain what kind of contingencies are we supporting and on which network element? A cross border element = tie lines and dangling lines? Can you be more precise?

Reply:
done